docs/configuration/prefill/objects_api.rst

```diff
@@ -0,0 +1,42 @@
+.. _configuration_prefill_objects_api:
+
+===========
+Objects API
+===========
+
+`Objects API`_ can store objects which are defined (their schema and properties) in the `Objecttypes API`_.
+With this prefill plugin we can have components/form fields which have pre-filled the values taken from the Objects API.
+
+.. note::
+
+   This service contains sensitive data and requires a connection to a specific
+   client system. The forms which make use of this prefill plugin require DigiD/Eherkenning authentication
+   of the user.
+
+.. _`Objects API`: https://objects-and-objecttypes-api.readthedocs.io/en/latest/
+.. _`Objecttypes API`: https://objects-and-objecttypes-api.readthedocs.io/en/latest/
+
+
+Configuration
+=============
+
+1. In Open Forms, navigate to: **Forms**
+2. Click **Add form**
+3. Define the necessary form details and add the desired components
+4. Navigate to: **Variables** tab
+5. Navigate to: **User defined** subtab
+6. Click **Add variable** and fill in the data from the available options:
+
+   * **Plugin**: *Choose the Objects API plugin*
+   * **API Group**: *Choose the desired API Group *
+     (There must be at least one added-configured via **Miscellaneous** > **Objects API groups**)
+   * **Objecttype**: *Based on the selected API group above, you can choose the objecttype from the auto populated list*
+   * **Mappings**: *On the left we have the available form variables and on the right the available attributes which are
+     defined in the selected objecttype's schema properties. You can do the mappings accordingly*
```

Comment:
```suggestion
6. Click **Add variable** and fill in the data from the available options:

   * **Plugin**: Choose the *Objects API* plugin
   * **API Group**: Select the appropriate API group. These API groups should be set up by an administrator,
     via **Admin** > **Configuration** > **Prefill plugins** > **Objects API** > **Manage API groups**
   * **Objecttype**: Select the expected object type from the dropdown.
   * **Mappings**: Configure which property from the Objects API record needs to be assigned to which form variable.
     For each form variable you want to pre-fill, add a new mapping. Then, on the left select the desired form variable,
     and on the right you can specify which property from the object type contains the value.
```

This will also require you to add the respective action(s) to the prefill plugin config actions :)

We usually do not refer to the "Miscellaneous" menu in the admin, instead you are supposed to get there via the Configuration overview page.